src/ServiceControl.Monitoring/Http/Diagrams/TimingsAggregator.cs

```diff
@@ -111,6 +64,7 @@ public class MonitoredEndpoint
         public string[] EndpointInstanceIds { get; set; }
         public MonitoredEndpointValues ProcessingTime { get; set; }
         public MonitoredEndpointValues CriticalTime { get; set; }
+        public MonitoredEndpointValues Retries { get; set; }
```

Comment:
So we abstract `IProvideEndpointInstanceMonitoringData` and `IProvideEndpointMonitoringData` implementations but still keep the properties in here, "knowing" what are `Retries`, `ProcessingTime`, `CriticalTime`. It seems incoherent to me. A dictionary-like approach, with implementations of `IProvide...` adding keys, would be better maybe?